packages/headless/src/features/commerce/pagination/pagination-slice.ts

```diff
@@ -57,4 +65,5 @@ export const paginationReducer = createReducer(
 
 function handlePaginationReset(state: CommercePaginationState) {
   state.page = 0;
+  state.perPage = 0;
```

Comment:
I think we should rather make it null by default

Reply:
(null would also be the default value in the state / mock state for perPage)

Reply:
Done in https://github.com/coveo/ui-kit/pull/3773/commits/c2dd21d1d99b1fd6eea6ef7d992b7fc55f014fda